### ISSUE TITLE
Distant Horizons mod gets disabled - compat issue

### ISSUE THREAD

**Issue by InsaanNotFound** [paulevsGitch/BetterEnd#467]
What does happen:
When you enter the End dimension almost on instant the renderer gets disabled with chat message 'Distant Horizons renderer has encountered an exception! Renderer is now disabled to prevent futher issues."

When does it happen:
Almost instantly after entering the dimension.

Log part containing the issue:
```
[18:31:28] [Render thread/INFO]: Adding dim [Dim = the_end, Region = MovabeGridRingList[-44,-44] 9*9[81]] to world Tests
[18:31:28] [Render thread/ERROR]: Renderer thrown an uncaught exception: 
java.lang.IllegalArgumentException: Color parameter outside of expected range: Blue
	at Not Enough Crashes deobfuscated stack trace.(1.18.2+build.3) ~[?:?]
	at java.awt.Color.testColorValueRange(Color.java:312) ~[?:?]
	at java.awt.Color.<init>(Color.java:397) ~[?:?]
	at java.awt.Color.<init>(Color.java:492) ~[?:?]
	at fabric.com.seibel.lod.common.wrappers.minecraft.MinecraftRenderWrapper.getFogColor(MinecraftRenderWrapper.java:149) ~[DistantHorizons-1.6.3a-1.18.2.jar:?]
	at com.seibel.lod.core.render.LodRenderer.getFogColor(LodRenderer.java:438) ~[DistantHorizons-1.6.3a-1.18.2.jar:?]
	at com.seibel.lod.core.render.LodRenderer.drawLODs(LodRenderer.java:316) ~[DistantHorizons-1.6.3a-1.18.2.jar:?]
	at com.seibel.lod.core.api.ClientApi.renderLods(ClientApi.java:272) ~[DistantHorizons-1.6.3a-1.18.2.jar:?]
	at net.minecraft.client.render.WorldRenderer.handler$edf000$renderChunkLayer(WorldRenderer:11479) ~[?:?]
	at net.minecraft.client.render.WorldRenderer.renderLayer(WorldRenderer) ~[?:?]
	at net.minecraft.client.render.WorldRenderer.render(WorldRenderer:1241) ~[?:?]
	at net.minecraft.client.render.GameRenderer.renderWorld(GameRenderer:1034) ~[?:?]
	at net.minecraft.client.render.GameRenderer.render(GameRenderer:813) ~[?:?]
	at net.minecraft.client.MinecraftClient.render(MinecraftClient:1122) ~[?:?]
	at net.minecraft.client.MinecraftClient.run(MinecraftClient:737) ~[?:?]
	at net.minecraft.client.main.Main.main(Main:236) [client-intermediary.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.13.3.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
[18:31:28] [Render thread/INFO]: [CHAT] §4§l§uERROR: Distant Horizons renderer has encountered an exception!
[18:31:28] [Render thread/INFO]: [CHAT] §4Renderer is now disabled to prevent futher issues.
[18:31:28] [Render thread/INFO]: [CHAT] §4Exception detail: java.lang.IllegalArgumentException: Color parameter outside of expected range: Blue
```

If you need me I can contact Distant Horizons devs unless you want to do so yourself. Thank you in advance
Sincerely, Nihil

**Comment by InsaanNotFound** [comment]
Alright then the issue gets send directly to Distant Horizons devs then. Thanks for response

**Comment by coolGi69** [comment]
If you want, you can now close this issue as it has been fixed in the upcoming release of DH (version 1.7) or on commit [0c031588](https://gitlab.com/jeseibel/minecraft-lod-mod/-/commit/0c031588aa17190629bae8479ee5614c7b29170d).\
Although the issue was originally cased by BCLib, they are doing something a bit weird here as they are multiplying the color values by 4 which caused us to have a crash when trying to read it ([here](https://github.com/paulevsGitch/BCLib/blob/14c668dcd03264541509d7ac770edc6e6bdcc614/src/main/java/ru/bclib/mixin/client/FogRendererMixin.java#L40) is that line that does that)

**Comment by InsaanNotFound** [comment]
Alright, good to know. Closing as fixed.
